tests/test_RFT.cpp

```diff
@@ -457,8 +457,8 @@ namespace {
 
         explicit Setup(const ::Opm::Deck& deck)
             : es    { deck }
-            , python{ }
             , sched { deck, es , python }
+            , python{ }
```

Comment:
This change is wrong.  The initialization of `sched` uses the `python` member.  To actually fix this you need to reorder the data members instead.

Reply:
indeed. i did not see that, sadly not caught in review.